pandas/core/apply.py

```diff
@@ -17,9 +17,11 @@
 from pandas.core.dtypes.generic import ABCSeries
 
 from pandas.core.construction import create_series_with_explicit_dtype
+from pandas.core.frame import DataFrame
```

Comment:
don't import at all, if you actually need these there are ways but not this

Reply:
What would be another way?

Reply:
`self.obj._constructor` - this maintains subclasses as well

Reply:
Found another way,  if its a series i am using the `Series` method `infer_objects`